packages/api-headless-cms/src/handler/plugins/graphqlFields/richText.ts

```diff
@@ -0,0 +1,66 @@
+
+import { CmsModelFieldToGraphQLPlugin } from "@webiny/api-headless-cms/types";
+
+const createListFilters = ({ field }) => {
+    return `
+        # Matches if the field is equal to the given value
+        ${field.fieldId}: JSON
+
+        # Matches if the field is not equal to the given value
+        ${field.fieldId}_not: JSON
+
```

Comment:
These list filters are not needed. We can't really filter by content. So, remove all.